src/App.jsx

```diff
@@ -45,7 +45,7 @@ function App() {
           <Route path="/" element={<Home />} />
           <Route path="/login" element={<Login />} /> 
           <Route path="/register" element={<Register />} /> 
-          <Route path="/posts" element={<PostsPage />} />
+          <Route path="/posts/" element={<PostsPage />} />
```

Comment:
Please don't add slash at the end
should be <Route path="/posts" element={<PostsPage />} />

Reply:
ok. Thanks